manager/src/main/java/com/redhat/service/bridge/manager/actions/connectors/ConnectorActionUtils.java

```diff
@@ -0,0 +1,38 @@
+package com.redhat.service.bridge.manager.actions.connectors;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.BooleanNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import static com.redhat.service.bridge.manager.actions.connectors.ConnectorAction.PROCESSORS_PARAMETER;
+
+public class ConnectorActionUtils {
```

Comment:
I like the fact that you've moved the logic in a separate class, but I would change the name.

`ConnectorProcessors` or `ConnectorSMT` rather than a generic `*util`

Reply:
This class was removed with the new refactoring